jOOQ/src/main/java/org/jooq/User.java

```diff
@@ -9,5 +9,5 @@
     /**
      * The name of user
      */
-    public String getName();
+    public Name getName();
```

Comment:
OK, I'll merge this but I'll change this back again. While the *internal* representation should be `Name`, the *external* one will be a `String` for API consistency reasons. Sorry... :)

Reply:
Okay. Could you explain why  the external representation to be `String`, please.

Reply:
For API consistency reasons with all the other API, e.g. `Table.getName()` or `Field.getName()`. Yes, it would be nice to return a `Name` reference here, but historically, we're returning strings...

Reply:
I see, thanks!